src/main/resources/db/migration/V203__deletingAndInsertingDataCorrectly.sql

```diff
@@ -0,0 +1,39 @@
+TRUNCATE genre, book, user_account, statistics CASCADE;
+
```

Comment:
Looks great. Just edit naming. I'm used to see it written in snake case and without -ing. Ex. delete_and_insert...